meilisearch-core/src/update/documents_addition.rs

```diff
@@ -289,6 +246,11 @@ pub fn reindex_all_documents(writer: &mut heed::RwTxn<MainT>, index: &store::Ind
     let mut indexer = RawIndexer::new(stop_words);
     let mut ram_store = HashMap::new();
 
+    if let Some(ref attributes_for_facetting) = index.main.attributes_for_faceting(writer)? {
+        let facet_map = facets::facet_map_from_docids(&index, &documents_ids_to_reindex, writer, &attributes_for_facetting)?;
+        index.facets.add(writer, facet_map)?;
+    }
+    // ^-- merge? --v
```

Comment:
What does this comment means?

Reply:
ah yes, for now I have factorized code to create the facet map in a separate function. However it could use some of the work done in the block under to save some calls to store. It's a performance vs readability/maintanability tradeoff. I left this comment here to remind us that a non obvious  merge is possible, if we decide that the performance gain is worth it.